host_os.py

```diff
@@ -34,6 +35,7 @@
     'build-package': build_package,
     'release-notes': create_release_notes,
     'upgrade-versions': upgrade_versions,
+    'update-versions-readme': update_versions_readme,
```

Comment:
I'd prefer to do this in upgrade-versions instead of creating a new subcommand. Both will always be executed together.